test/e2e-rekt-tests.sh

```diff
@@ -50,4 +50,10 @@ kubectl apply -Rf "$(dirname "$0")/config-authentication-oidc"
 
 go_test_e2e -timeout=1h ./test/rekt -run OIDC || fail_test
 
+echo "Running E2E Authorization Reconciler Tests"
+
+kubectl apply -Rf "$(dirname "$0")/config-authentication-oidc"
```

Comment:
This is applied a few lines before, if we run both OIDC and authz tests, I think it's even faster

Reply:
I mostly did this to have them separated for overview. But we can also run them together with the OIDC tests...

Reply:
updated